frontend/packages/pipelines-plugin/src/components/pipelines/list-page/__tests__/PipelineAugmentRunsWrapper.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { Firehose } from '@console/internal/components/utils';
+import { ListPageWrapper_ as ListPageWrapper } from '@console/internal/components/factory';
+import { PipelineExampleNames, pipelineTestData } from '../../../../test-data/pipeline-data';
+import PipelineAugmentRunsWrapper from '../PipelineAugmentRunsWrapper';
+import PipelineAugmentRuns from '../PipelineAugmentRuns';
+
+const mockData = pipelineTestData[PipelineExampleNames.WORKSPACE_PIPELINE];
+const { pipeline } = mockData;
+
+type PipelineAugmentRunsWrapperProps = React.ComponentProps<typeof PipelineAugmentRunsWrapper>;
+
+describe('Pipeline Augment Run Wrapper', () => {
+  let pipelineAugmentRunsWrapperProps: PipelineAugmentRunsWrapperProps;
+  let wrapper: ShallowWrapper;
+  beforeEach(() => {
+    pipelineAugmentRunsWrapperProps = {
+      pipeline: {
+        data: [pipeline],
+        loaded: false,
+      },
+    };
+    wrapper = shallow(<PipelineAugmentRunsWrapper {...pipelineAugmentRunsWrapperProps} />);
+  });
+
+  it('Should not render if the firehose call is not yet loaded', () => {
+    expect(wrapper.html()).toBeNull();
+  });
+
+  it('Should render No Pipelines found text if the data is empty', () => {
+    wrapper.setProps({ pipeline: { data: [], loaded: true } });
+    expect(wrapper.find('div.text-center').text()).toBe('No Pipelines Found');
```

Comment:
🤔  I am not happy with these kind of tests... they are brittle and tie themselves to the exact implementation. If we swapped the divs out for PF components, this test would fail, but effectively not useful to us in anyway. If we changed the text, that would also not be ideal.

Perhaps the test here should be you don't render the ListPage 🤔